depthai_ros_driver/launch/depthai_node.launch

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="ns" default="depthai"/>
+  <arg name="node_name" default="driver"/>
+
+  <arg name="queue_size" default="1"/>
+
+  <arg name="camera_name" default="bw1097"/>
+  <arg name="camera_param_uri" default="package://depthai_ros_driver/params/camera/"
+    description="Location of parameters is {camera_param_uri}/{camera_name}/{stream}.yaml"/>
```

Comment:
`description` -> `doc`
same for `depthai_nodelet.launch`

Reply:
Done. PTAL